src/app/games/quixo/QuixoRules.ts

```diff
@@ -17,6 +17,8 @@ import { MGPMap } from 'src/app/utils/MGPMap';
 import { NumberConfig, RulesConfigDescription, RulesConfigDescriptionLocalizable } from 'src/app/components/wrapper-components/rules-configuration/RulesConfigDescription';
 import { MGPValidators } from 'src/app/utils/MGPValidator';
 import { TableUtils } from 'src/app/utils/ArrayUtils';
+import { NumberMap } from 'src/app/utils/NumberMap';
```

Comment:
ERROR Error: Assertion failure: Invalid direction: pawn on the bottom side can't be moved down.
cf. screenshot
![2024-01-21_11-09](https://github.com/EveryBoard/EveryBoard/assets/58877/ade95de7-6980-4e1b-97c8-3c1e780bd564)

Reply:
Eeeeh revieuweur de paaaaitte, c'est CONFIGURE qu'a cassé ça :D !!!